crates/uv/tests/it/show_settings.rs

```diff
@@ -10,12 +10,16 @@ use crate::common::{uv_snapshot, TestContext};
 /// In particular, remove any user-defined environment variables and set any machine-specific
 /// environment variables to static values.
 fn add_shared_args(mut command: Command) -> Command {
+    // Create an empty directory for XDG_CONFIG_DIRS to avoid loading system configuration.
+    let xdg = assert_fs::TempDir::new().expect("Failed to create temp dir");
```

Comment:
It's a little strange because this directory will be dropped and deleted as soon as the method returns.

Reply:
That's what I get for doing guesswork copy&paste. I guess it was happy with any value, whether it existed or not. I'll test your version in a minute, just lemme finish with triton first, since my home is not large enough for uv and something else :-).

Reply:
Well, if I were to go lawyering:

> If $XDG_CONFIG_DIRS is either not set or empty, a value equal to /etc/xdg should be used.

It doesn't say anything about non-existing, so I guess any value is sufficient to disable `/etc/xdg`. Still, I guess it's safer to use a real directory here to avoid any future issues.